src/lib/component/base/RelativeDate/RelativeDate.test.js

```diff
@@ -1,40 +1,47 @@
 import React from "/vendor/react";
-import { mount } from "enzyme";
 import IntlRelativeFormat from "intl-relativeformat";
+import { render } from "react-testing-library";
+
 import RelativeDate from "./RelativeDate";
 
 global.IntlRelativeFormat = IntlRelativeFormat;
 
 test("RelativeDate given date seconds ago", () => {
   const now = new Date();
   const date = new Date(now.getTime() - 12000);
-  const elem = mount(<RelativeDate to={now} dateTime={date.toUTCString()} />);
+  const { getByText } = render(
+    <RelativeDate to={now} dateTime={date.toUTCString()} />,
+  );
 
-  expect(elem.text()).toEqual("seconds ago");
+  expect(getByText("seconds ago"));
```

Comment:
This seems a bit strange, do we not want to have it match a value?

Reply:
I think I can improve the matcher I'm using here.